src/Tools/babylon.database.ts

```diff
@@ -530,7 +530,7 @@ module BABYLON {
                                 try {
                                     //backwards compatibility with ts 1.0, srcElement doesn't have an "error" according to ts 1.3
                                     if (event.srcElement['error'] && event.srcElement['error'].name === "QuotaExceededError") {
-                                        this.hasReachedQuota = true;
+                                        (this as any).hasReachedQuota = true;

```

Comment:
it compile with a lambda, otherwise we can keep a function and declare "var self = this" outside the function and use "self" instead of this and it will also compile